docs/application/dotnet/tutorials/certificates/managing-certificate-profile.md

```diff
@@ -0,0 +1,83 @@
+# Managing Certificate Profile
+
+Before installing your application on a device or submitting it to the Tizen Store, it must be signed with a certificate profile.
+The signature verifies the source of the application and makes sure it has not been tampered with since its publication.
+A certificate profile is a combination of the certificates used for signing.
+
+## General Information
+
+In the menu bar, click `Tools > Tizen > Tizen Certificate Manager` to launch the Certificate Manager.
+
+![Tizen Certificate Manager with two profile](media/certification_guide18.png)
+
+* The Active profile is indicated by the check mark.
+* All of the application in this SDK will be signed with this active profile.
+* You can create, delete, and set a profile to active here.
+* Lower window shows the information of each certificate.
+* You can replace the certificate with another existing one.
+* The Tizen platform allows multiple distributor certificates up to 2 which supports other device manufacturers or telecommunication companies but it is not needed for most of developers generally.
+
+## Changing Only a Distributor Certificate in a Profile
+
+There might be an instance where you need to register more devices or change the devices to the distributor certificate.  
+However author certificates rarely needs to be changed.  
+This section describes how to do it.
+
+Do the same steps to create a certificate profile described in Creating Certificates before step [Create new or select old profile](creating-certificates.md#4-create-new-or-select-old-profile)
+
+### 1. Select an existing certificate profile
+
+Click `Select an existing certificate profile` and select a profile which you want to change the registered device then click `Next`.
+
+![Select an existing certificate profile](media/certification_guide19.png)
+
+### 2. Use the existing author certificate
+
+Click `No` to use the existing author certificate then it moves to [Create new or select existing distributor certiicate](creating-certificates.md#1-create-new-or-select-existing-distributor-certiicate) to register DUID.
+
+![Alert popup to remove author certificate](media/certification_guide20.png)
+
+> If you click `Yes`, it moves to [Create a new author certificate](creating-certificates.md#create-a-new-author-certificate) step and original author certificate will be removed permanently.
+> So it should be carefully considered.  
+> However the SDK automatically backups the original author.p12 file with timestamp extension like author.p12_bak_20160806092013 in the profile directory to protect developer from unintended removal.
+
+### 3. Create new Distributor for certificate profile
+
+Next steps are same as guided in [Create Certificate](creating-certificates.md) after [Create new or select existing distributor certiicate](creating-certificates.md#1-create-new-or-select-existing-distributor-certiicate) step
```

Comment:
Follow the same steps as mentioned in [creating certificate](creating-certificates.md) after [creating new or selecting existing distributor certificate](creating-certificates.md#1-create-new-or-select-existing-distributor-certificate).

Is it fine to change hyperlink text?
Please verify whether the technical meaning is altered.

Reply:
done